extensions/blocks/contact-form/components/jetpack-field.js

```diff
@@ -35,7 +30,28 @@ export default function JetpackField( props ) {
 				</Disabled>
 			</div>
 
-			<JetpackFieldControls id={ id } required={ required } setAttributes={ setAttributes } />
-		</Fragment>
+			<JetpackFieldControls
+				id={ id }
+				required={ required }
+				width={ width }
+				setAttributes={ setAttributes }
+			/>
+		</>
 	);
 }
+
+const withCustomClassName = createHigherOrderComponent( BlockListBlock => {
+	return props => {
+		if ( props.name.indexOf( 'jetpack/field' ) > -1 ) {
+			const customClassName = props.attributes.width
+				? 'jetpack-field__width-' + props.attributes.width
+				: '';
+
+			return <BlockListBlock { ...props } className={ customClassName } />;
+		}
+
+		return <BlockListBlock { ...props } />;
+	};
+}, 'withCustomClassName' );
+
+addFilter( 'editor.BlockListBlock', 'jetpack/contact-form', withCustomClassName );
```

Comment:
Ugh, I guess a filter is the only way to add a class to the wrapper, isn't it? 😞

Reply:
Yep.... could not find any other way.